src/main/java/InputParser.java

```diff
@@ -1,13 +1,31 @@
 public class InputParser {
     static Object[] parseInput(String input) {
         InputType type = InputType.NONE;
-        String value = input;
+        String[] value = new String[]{ input };
         for(InputType inputType: InputType.values()) {
+            if (inputType == InputType.NONE) {
+                continue;
+            }
+
             if (input.startsWith(inputType.label)) {
-                value = "";
-                if (inputType.label.length() < input.length()) {
-                    value = input.substring(inputType.label.length() + 1);
+                if (inputType == InputType.BYE || inputType == InputType.LIST) {
```

Comment:
Maybe it would be neater to separate out into boolean variables for example `isBye`, `isList`, etc to be in line with the code quality guideline